apps/notepad/src/app/component/notes-list/notes-list.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { Note } from './note.model';
+
+@Component({
+  selector: 'np-notepad-input',
+  templateUrl: './notes-list.component.html',
+  styleUrls: ['./notes-list.component.scss'],
+})
+export class NotesListComponent implements OnInit {
+  notesList: Note[] = [];
+
+  ngOnInit() {
+    const initialList = localStorage.getItem('storelist');
```

Comment:
If notes are being fetched from localStorage at multiple places, then a common service file can be made for code reusability in multiple components.